src/2023/index.html

```diff
@@ -63,68 +63,84 @@ <h2 class="highlight eyebrow">The first Swiss Ruby conference</h2>
           <h2>Speakers</h2>
           <div class="grid speakers center">
             <article class="stack speaker-preview">
-              <figure>
-                <img class="speaker-picture" width="200" height="200" alt="Vladimir Dementyev's speaker picture" src="images/speakers/vladimir-dementyev.jpeg">
-              </figure>
-              <div>
-                Vladimir Dementyev
-              </div>
+              <a href="talks#vladimir-dementyev-profiling-ruby-tests">
+                <figure>
+                  <img class="speaker-picture" width="200" height="200" alt="Vladimir Dementyev's speaker picture" src="images/speakers/vladimir-dementyev.jpeg">
+                </figure>
+                <div>
+                  Vladimir Dementyev
+                </div>
+              </a>
             </article>
             <article class="stack speaker-preview">
-              <figure>
-                <img class="speaker-picture" width="200" height="200" alt="Isabel Carbotta's speaker picture" src="images/speakers/isabel-carbotta.jpeg">
-              </figure>
-              <div>
-                Isabel Carbotta
-              </div>
+              <a href="talks#isabel-carbotta-how-to-bootstrap-your-software-startup">
```

Comment:
nitpick: the id could be maybe simplified just to the speakers name. But now it would be additional work to change it so no need to do it, maybe something to keep in mind in the future.

Reply:
Right now we only link from the speakers list. In the future, we would also link to the same place from the schedule page, where perhaps the talk is more important. I think the speaker and talk are inseparable, which is why I put both in the id and link to that entire section. What do you think?